WeightProducer/python/MCSampleValues.py

```diff
@@ -251,669 +247,501 @@ class MCSampleValuesHelper():
+                XS_13TeV=1554000.0, XSSource_13TeV="XSDB (LO)",
             ),
         },
-        "QCD_Pt-470to600_MuEnrichedPt5" : {
+        "QCD_HT300to500" : {
             "CrossSection" : XSValues(
-                XS_2016=79.25, XSSource_2016="GenXSecAnalyzer",
-                XS_2017=59.24, XSSource_2017="XSDB",
-                XS_2018=59.06, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=325000.0, XSSource_13TeV="GenXSecAnalyzer",
             ),
         },
-        "QCD_Pt-600to800_MuEnrichedPt5" : {
+        "QCD_HT500to700" : {
             "CrossSection" : XSValues(
-                XS_2016=25.25, XSSource_2016="XSDB",
-                XS_2017=18.18, XSSource_2017="GenXSecAnalyzer",
-                XS_2018=18.21, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=30350.0, XSSource_13TeV="GenXSecAnalyzer",
             ),
         },
-        "QCD_Pt-800to1000_MuEnrichedPt5" : {
+        "QCD_HT700to1000" : {
             "CrossSection" : XSValues(
-                XS_2016=4.723, XSSource_2016="XSDB",
-                XS_2017=3.277, XSSource_2017="GenXSecAnalyzer",
-                XS_2018=3.276, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=6403.0, XSSource_13TeV="GenXSecAnalyzer",
             ),
         },
-        "QCD_Pt-1000toInf_MuEnrichedPt5" : {
+        "QCD_HT1000to1500" : {
             "CrossSection" : XSValues(
-                XS_2016=1.613, XSSource_2016="XSDB",
-                XS_2017=1.079, XSSource_2017="GenXSecAnalyzer",
-                XS_2018=1.079, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=1117.0, XSSource_13TeV="GenXSecAnalyzer",
             ),
         },
-        "QCD_Pt-15to7000_Flat" : {
+        "QCD_HT1500to2000" : {
             "CrossSection" : XSValues(
-                XS_2016=1976000000.0, XSSource_2016="XSDB",
-                XS_2017=1370000000.0, XSSource_2017="XSDB",
-                XS_2018=1371000000.0, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=108.4, XSSource_13TeV="GenXSecAnalyzer",
             ),
         },
-        "QCD_Pt-15to7000_FlatP6" : {
+        "QCD_HT2000toInf" : {
             "CrossSection" : XSValues(
-                XS_2016=1973000000.0, XSSource_2016="XSDB",
+                XS_13TeV=21.93, XSSource_13TeV="XSDB (LO)",
             ),
         },
-        "QCD_Pt-15to7000_Flat2017" : {
-            "CrossSection" : XSValues(
-                XS_2017=1361000000.0, XSSource_2017="XSDB",
+        "QCD_Pt_15to30" : {
+           "CrossSection" : XSValues(
+                 XS_13TeV=1244000000.0, XSSource_13TeV="XSDB (unknown)",
             ),
         },
-        "QCD_Pt-15to7000_Flat2018" : {
-            "CrossSection" : XSValues(
-                XS_2018=1372000000.0, XSSource_2018="GenXSecAnalyzer",
+        "QCD_Pt_30to50" : {
+           "CrossSection" : XSValues(
+                 XS_13TeV=106500000.0, XSSource_13TeV="XSDB (unknown)",
             ),
         },
         "QCD_Pt_50to80" : {
```

Comment:
Do you not need the MuEnriched samples anymore?

Reply:
Yeah, when reviewing with @cmadrid1 the samples the Stealth/RPV group has been using, we do not see a need for them going forward.